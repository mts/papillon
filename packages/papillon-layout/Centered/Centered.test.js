import centered from './Centered.render';

describe('<Layout />', () => {
  let wrapper;

  describe('Instance', () => {
    test('must be an instance of Centered', () => {
      wrapper = renderMount(centered);
      expect(wrapper.find('Centered')).toHaveLength(1);
    });
  });

  describe('Snaphot', () => {
    test('must match centered', () => {
      expect(global.renderToJSON(centered)).toMatchSnapshot();
    });
  });
});
