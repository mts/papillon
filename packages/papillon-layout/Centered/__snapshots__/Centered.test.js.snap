// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> Snaphot must match centered 1`] = `
<div
  className="columns"
>
  <div
    className="one-half column centered block-blue border p-3"
  >
    .one-half
  </div>
</div>
`;
